graphcanvas/tests/test_layout.py

```diff
@@ -9,7 +10,8 @@ class TestLayout(unittest.TestCase):
     def assert_layout_positions(self, result, expected):
         result_x, result_y = result
         expected_x, expected_y = expected
-        self.assertAlmostEqual(result_x, expected_x)
+        _, mod_x = divmod(result_x, expected_x)
+        self.assertAlmostEqual(mod_x, 0.0)
```

Comment:
I don't recall precisely why I did this. I think there was some issue I was seeing with non-deterministic ordering of the nodes horizontally, or that the ordering was somehow different between py27 and py35 (maybe the problem showed up when I was testing with py34 at first?). I don't see the problem now, though, so I will revert the change.